docs/tutorials/best-practices/security-best-practices.md

```diff
@@ -0,0 +1,496 @@
+# Best Practices for Coder Security
+
+December 30, 2024
+
+---
+
+This best practices guide is separated into parts to help you secure your Coder
+deployment through understanding each threat model, hardening authentication,
+and encryption.
+
+As with any security guide, the steps and suggestions outlined in this document
+are not meant to be exhaustive and do not offer any guarantee.
+
+## Coder Server
+
+Coder Server is the main control core of a Coder deployment.
+
+If the Coder Server is compromised in a security incident, it can affect every
+other part of your deployment. Even a successful read-only attack against the
+Coder Server could result in a complete compromise of the Coder deployment if
+credentials are stolen.
+
+### User authentication
+
+Configure [OIDC authentication](../../admin/users/oidc-auth.md) against your
+organization’s Identity Provider (IdP), such as Okta, to allow single-sign on:
```

Comment:
```suggestion
organization’s Identity Provider (IdP), such as Okta, to allow single-sign on.
```

Following list is additional recommendations, not instructions on setting up single sign-on